src/fides/api/alembic/migrations/versions/4fb779cc5f17_add_last_monitored_and_enabled_to_monitor_config.py

```diff
@@ -0,0 +1,36 @@
+"""Add last_monitored and enabled to MonitorConfig
+
+Revision ID: 4fb779cc5f17
+Revises: 7641ea685ea4
+Create Date: 2024-06-20 13:45:46.024496
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "4fb779cc5f17"
+down_revision = "7641ea685ea4"
```

Comment:
just a heads up, i _think_ you'll need to update this down rev as there was a migration merged yesterday evening in https://github.com/ethyca/fides/commit/2ae70bbbdf207005cb8cccbbcc0147747634c73f#diff-6c7a7d29f3aba25735c55f0c99cfca6eb253be8ecca2e4d7bb4ba5ebc646d387

Reply:
Fixed in [07452cf](https://github.com/ethyca/fides/pull/4991/commits/07452cf3815b44a9474036895075bbf28e4f8806)